src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -61,12 +64,19 @@ public Events(FlareBot bot) {
     @Override
     public void onReady(ReadyEvent event) {
         FlareBot.getInstance().latch.countDown();
+        new Timer().schedule(new TimerTask() { //If you're not ok with this in the ready event just say so.
+            @Override
+            public void run() {
+                spamMap = new HashMap<>(); //According to stack overflow it's better to just create a new hashmap and let the garbage collector get the other one.
```

Comment:
That shouldn't be called expiring then but this is bad for ~~two~~three reasons: 
* There is a case which allows spam to come through if timed properly
* The map shouldn't just be thrown away
* The map is not concurrent